src/Compilers/CSharp/Portable/Symbols/Source/SourceParameterSymbol.cs

```diff
@@ -222,10 +222,21 @@ public sealed override RefKind RefKind
             }
         }
 
-        internal sealed override DeclarationScope DeclaredScope => _scope;
+        /// <summary>
+        /// The declared scope. From source, this is from the <c>scope</c> keyword only.
+        /// </summary>
+        internal DeclarationScope DeclaredScope => _scope;
```

Comment:
>DeclaredScope [](http://example.com/codeflow?start=34&length=13)

It looks like this fixes [https://github.com/dotnet/roslyn/issues/63078](https://github.com/dotnet/roslyn/issues/63078).

Reply:
> It looks like this fixes #63078.

Thanks. I'll mark the issue as closed by this PR.